docs/source/integrations/building-integrations.md

```diff
@@ -196,10 +151,11 @@ interface HTTPGraphQLRequest {
 }
 ```
 
-We now have the `method`, `body`, and `search` properties computed. The only
-thing left for us to compute is the `headers` object! Apollo Server expects a
-`Map` of headers. In the Express implementation, we construct the `Map` by
-iterating over the `headers` object like so:
+Apollo Server handles the logic of extracting the HTTP method (i.e., `GET` vs. `POST`), relevant headers, and it knows whether to check the `body` or `search` fields for the GraphQL-specific parts of a query. So, Apollo Server handles your `method`, `body`, and `search` properties for the `HTTPGraphQLRequest`.
+
+Finally, Apollo Server expects the `headers` property to return a `Map` of headers.
```

Comment:
Specifically the word `return` here implies that headers is a function that returns a Map, when it is in fact just a Map itself.
```suggestion
Finally, Apollo Server expects the `headers` property to be a `Map` of headers.
```